org.nongnu.gsequencer.gsequencer.json

```diff
@@ -154,11 +156,41 @@
                 }
             ]
         },
+        {
+            "name": "vst3sdk",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "build-options": {
+                "env": {
+                    "CXXFLAGS": "-include cstdio"
+                }
+            },
+            "build-commands": [
+                "cmake -DCMAKE_BUILD_TYPE=RelWithDebugInfo ..",
+                "ninja",
```

Comment:
```suggestion
            "post-install": [
```

Reply:
you have buildsystem = ninja so build-commands is ignored. (there should even be a warning)